core/src/main/java/discord4j/core/DiscordClientBuilder.java

```diff
@@ -102,6 +112,16 @@ public DiscordClientBuilder setShardCount(final int shardCount) {
         return this;
     }
 
+    public StoreServiceLoader getStoreServiceLoader() {
```

Comment:
If we're adding this to the builder, we should remove ``getStoreService`` and ``setStoreService`` methods, or at least avoid duplicating the ways StoreService can be customized.

Reply:
I guess this is superfluous and can be removed